### PR TITLE
[OTX][Unit-Test] Add unit tests for otx.algorithms.action.adapters.mmaction.data

### PR DESCRIPTION
**Pull request by jaegukhyun**
This PR is for adding unit tests which check otx.algorithms.action.adapters.mmaction.data

Coverage Report:
image

I guess we can discuss about general format of unit-test

### CODE REVIEW

File: tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        2. Get sample inputs from sample OTXActionClsDataset
+        3. Add "frame_inds", "gt_bboxes", "proposals" attributes to sample inputs
+        4. Check RawFrameDecode transform's results
+            1. Whether transform creates imgs
+            2. Whether transform creates proper img size
+            3. Whether transform modify gt_bboxes and proposals w.r.t img_size
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self) -> None:
```
Comment (eunwoosh) [inline tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py:40]: I think some of codes are duplicated across the test files. how about making common things as a single fixture and use it?

File: tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    @e2e_pytest_unit
+    def test_call(self):
+        """Test __call__ function."""
+
+        inputs = self.dataset[0]
+        inputs["frame_inds"] = list(range(2))
+        inputs["gt_bboxes"] = np.array([[0, 0, 1, 1]])
+        inputs["proposals"] = np.array([[0, 0, 1, 1]])
+        outputs = self.decode(inputs)
```
Comment (eunwoosh) [inline tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py]: How about initializing RawFrameDecode class in the test code and use it to increase readability? Actually I took some time to know which function is tested.

File: tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py (hunk 1 above)
Comment (jaegukhyun) [reply tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py:40]: I added test_helpers.py for duplicated codes. Thank you for your comment

File: tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py (hunk 2 above)
Comment (jaegukhyun) [reply tests/unit/algorithms/action/adapters/mmaction/data/pipelines/test_action_loading.py]: I moved the initialization code to test. Thanks

**Review by harimkang — CHANGES_REQUESTED** [review]
According to the CI test, 1 error occurred (action unit-test).
image
Could you please check it out?

**Comment by jaegukhyun** [comment]
After, rebase all the errors disappear
image

**Review by eunwoosh — DISMISSED** [review]
LGTM. Thanks for your work!

**Review by harimkang — DISMISSED** [review]

**Review by eunwoosh — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**